src/runner.js

```diff
@@ -49,6 +49,8 @@ export default async function run(options, forCli = false) {
     } else options.confidenceSet = confidence[options.confidenceSet.toUpperCase()];
   } else options.confidenceSet = confidence["TENTATIVE"]; // default to lowest
 
+  options.customScan = (options.customScan || []).map(c => c.toLowerCase());
```

Comment:
Wasn't this already done in:
https://github.com/doyensec/electronegativity/blob/81ae713c0112675e5550146c1b7380f7af4234ba/src/finder/finder.js#L25
and:
https://github.com/doyensec/electronegativity/blob/b7a40b3693f3ade87191f80df5c4c327f6465a53/src/finder/globalchecks.js#L24
Or is it different?

Reply:
The CLI version does the same thing:

https://github.com/doyensec/electronegativity/blob/master/src/index.js#L54

Reply:
You are right, I remember writing this check for CLI but I couldn't recall where. LGTM!